src/org/labkey/trialshare/TrialShareController.java

```diff
@@ -415,14 +415,14 @@ public void setNotificationSettings(boolean notificationSettings)
             this.notificationSettings = notificationSettings;
         }
 
-        public boolean getSampleTypes()
+        public boolean getSampleTypesAndDataClasses()
```

Comment:
do we anticipate wanting this setting to be separable or distinct to just sampleTypes or dataClasses?